ipapython/admintool.py

```diff
@@ -309,7 +312,7 @@ def log_failure(self, error_message, return_value, exception, backtrace):
         if error_message:
             logger.error('%s', error_message)
         message = "The %s command failed." % self.command_name
-        if self.log_file_name and return_value != 2:
+        if self.log_file_initialized and return_value != 2:
             # magic value because this is common between server and client
             # but imports are not straigthforward
             message += " See %s for more information" % self.log_file_name
```

Comment:
I tested CTRL-C during the backup launched with `ipa-backup --log-file /tmp/foo`, and the error message always refers to `See /var/log/ipabackup.log for more information` even if a different log file was used. The var self.log_file_name is never assigned self.options.log_file, this should probably be done in __setup_logging.

Reply:
Good catch, this seems to be a long-standing bug on its own!